src/api/ledger/accountinfo.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const utils = require('./utils');
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+
+function formatAccountInfo(response) {
+  const data = response.account_data;
+  return {
+    sequence: data.Sequence,
+    xrpBalance: utils.common.dropsToXrp(data.Balance),
+    ownerCount: data.OwnerCount,
+    previousTransactionID: data.PreviousTxnID,
+    previousTransactionLedgerVersion: data.PreviousTxnLgrSeq
```

Comment:
PreviousTxnID is always set in the account root, it's part of [threading](https://wiki.ripple.com/Transaction_Metadata#Threading) (link refers to metadata but it's a ledger entry thing).  The rules are basically (actually more nuanced), that PreviousTxnID  gets updated for ANY txn that affects the account, and that AccountTxnID (if enabled via AccountSet asfAccountTxnID) is updated with the id of every Account initiated txn.